code/game/objects/items/lock.dm

```diff
@@ -0,0 +1,84 @@
+/obj/item/lock_construct
+	name = "\improper lock"
+	icon = 'icons/obj/lock.dmi'
+	w_class = WEIGHT_CLASS_SMALL
+	var/lock_data
+	var/static/lock_uid = 1
+	var/locked = FALSE
+	var/prying = FALSE //if somebody is trying to pry us off
+
+/obj/item/lock_construct/New() // Same system machines use for UID. Could probably add a global UID for everything if you wanted and use it for shenanigans, or simpler loading.
```

Comment:
initialize()

Reply:
Isn't initialize for mapload, and new is for any time an object is created?

Reply:
initialize is a controlled new, so instead of byond handling it, it's the MC